lib/components/src/highlight_button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import glamorous from 'glamorous';
+
+const Button = props => <span role="button" tabIndex={-1} {...props} />;
```

Comment:
Why `span` not `button`?

Reply:
I did not want to have to reset/restyle a `button`

Reply:
Native buttons have some advantages like automatically triggering click event on `Enter`. I believe it’s worth a bit of extra styling